src/middlewared/middlewared/utils/directoryservices/krb5.py

```diff
@@ -96,6 +96,17 @@
 }
 
 
+def get_persistent_keyring_session(uid: int = 0) -> int:
```

Comment:
Some docs would be nice.

Reply:
IIRC I did some further testing and realized I didn't actually need to do this and so I can back out.